app/src/main/java/com/easyfitness/SettingsFragment.java

```diff
@@ -128,6 +147,13 @@ public void onCreate(Bundle savedInstanceState) {
         });
     }
 
+    private String getDate(long milliSeconds) {
+        SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss");
```

Comment:
This should be localized. Date format is different depending on countries. 
You can use function  dateToLocalDateStr in utils/DataConverter and then add the time.
Thanks.

Reply:
Solved by 966ab3c696d62518e082eb1c7265460ca87e6627. 

I concatenated the String of `dateToLocalDateStr` with the time having the format "HH:mm:ss" to display also the time since the function `dateToLocalDateStr` returns only the date. Is that fine for you?